### DISCUSSION TITLE
L1-0x00 Nervos(CKB) 极速上手

### DISCUSSION THREAD

**Discussion by leeduckgo** [NonceGeek/Move-Camp#19] [Amazing Things!]
- Hello, CKB! — CKB 本地节点搭建简明指南
- Nervos Chain 介绍
- 智能合约学习工具介绍
- Set-purpose 合约源码讲解

**Comment by leeduckgo** [comment]
基础 setPurpose 合约：
> https://gist.github.com/leeduckgo/b6415bcb9a694d2f66fc5e4e65d63af2

作业（二选一）: 

> 1. 查阅相关资料，将直接设置`owner`改为通过引入`ownerable.sol`。
> 2. 查阅资料，在 `setPurpose` 函数中放入`emit`语句，以通过 event 的方式记录`set-purpose`的历史。

作业打卡：

在本帖下留下完成合约的截图即可。

**Comment by Az0s** [comment]
打卡～
作业2: event log
新建event保存发送地址、新purpose值和交易value
<img width="1391" alt="Screen Shot 2021-12-14 at 10 31 12 AM" src="https://user-images.githubusercontent.com/42002876/145922423-b3585033-28c8-4232-a304-73ea43b8a366.png">
